react/components/Header/Summary.js

```diff
@@ -25,56 +26,61 @@ const OrderSummary = ({ data, intl }) => {
   return (
     totalDeliveries.length > 0 &&
     totalPickUps.length > 0 && (
-      <section className="w-80 center">
+      <section className="w-80 center bb b--muted-4 pb8">
         <PageBlock variation="half">
-          <article className="mb8 center">
-            <p className="t-heading-4 tc">
-              {intlMessage(intl, 'summary.shipping')}
-            </p>
-            <hr className="bg-muted-4 bt b--muted-4" />
-            <p>
-              {intlMessage(intl, 'summary.items', {
-                itemsQuantity: deliveryItemsQuantity,
-              })}
-              {intlMessage(intl, 'summary.shipping.quantity', {
-                shippings: totalDeliveries.length,
-              })}
-            </p>
-            <p className="t-heading-4">
-              <TranslateEstimate
-                shippingEstimate={longestDeliveryEstimate.shippingEstimate}
-              />
-            </p>
-            <small className="t-small c-muted-2">
+          <article className="center flex flex-column">
```

Comment:
Useless `flex` & `flex-column` classes. Even if the direct child `div` is removed, `p` elements are block elements, therefore they will stack vertically.

Reply:
Yep, removed them.